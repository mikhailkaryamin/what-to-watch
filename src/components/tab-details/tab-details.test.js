import React from 'react';
import renderer from 'react-test-renderer';

import TabDetails from './tab-details.tsx';

describe(`Tab details`, () => {
  const DIRECTOR = `big boss`;
  const GENRE = `comedy`;
  const RELEASED = 2005;
  const RUN_TIME = 200;
  const STARRING = [
    `Mike`,
    `Sam`,
    `John`,
  ];

  const wrapper = renderer
    .create(
        <TabDetails
          director={DIRECTOR}
          genre={GENRE}
          released={RELEASED}
          runTime={RUN_TIME}
          starring={STARRING}
        />
    ).toJSON();

  test(`should render tab details`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
