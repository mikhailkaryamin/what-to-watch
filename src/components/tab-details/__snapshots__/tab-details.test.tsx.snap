// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab details should render tab details 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        big boss
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Mike
        , 
         
        <br />
        Sam
        , 
         
        <br />
        John
        
         
        <br />
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        3h 20m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        comedy
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2005
      </span>
    </p>
  </div>
</div>
`;
